content/ko/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -0,0 +1,79 @@
+---
+# reviewers:
+# - jayunit100
+# - jsturtevant
+# - marosset
+# - perithompson
+title: 윈도우 노드의 자원 관리
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+이 페이지는 리눅스와 윈도우 간에 리소스를 관리하는 방법의 차이점을 간략하게 설명한다.
+
+<!-- body -->
+
+리눅스 노드에서, {{< glossary_tooltip text="cgroup" term_id="cgroup" >}}이 
+리소스 제어를 위한 파드 경계로서 사용된다. 
+컨테이너는 네트워크, 프로세스 및 파일시스템 격리를 위해 해당 경계 내에 생성된다. 
+cpu/io/memory 통계를 수집하기 위해 cgroup API를 사용할 수 있다.
+
+반대로, 윈도우는 시스템 네임스페이스 필터와 함께 
+컨테이너별로 [잡(job) 오브젝트](https://docs.microsoft.com/windows/win32/procthread/job-objects)를 사용하여 
+모든 프로세스를 컨테이너 안에 포함시키고 호스트와의 논리적 격리를 제공한다.
+(잡 오브젝트는 윈도우의 프로세스 격리 메커니즘이며 
+쿠버네티스의 {{< glossary_tooltip term_id="job" text="잡(Job)" >}}과는 다른 것이다.)
+
+네임스페이스 필터링 없이 윈도우 컨테이너를 실행할 수 있는 방법은 없다. 
+이는 곧 시스템 권한이 호스트 컨텍스트에서 삽입될(assert) 수 없으며, 
+이로 인해 특권을 가진(privileged) 컨테이너는 윈도우에서 사용할 수 없음을 의미한다. 
+또한 보안 계정 매니져(Security Account Manager, SAM)가 분리되어 있으므로 
+컨테이너는 호스트의 ID를 가정(assume)할 수 없다.
+
+## 메모리 관리 {#resource-management-memory}
+
+윈도우에는 리눅스에는 있는 메모리 부족 프로세스 킬러가 없다. 
+윈도우는 모든 사용자-모드 메모리 할당을 항상 가상 메모리처럼 처리하며, 페이지파일이 필수이다.
+
+윈도우 노드는 프로세스를 위해 메모리를 오버커밋(overcommit)하지 않는다. 
+이로 인해 윈도우는 메모리 부족(OOM, out of memory) 컨디션에 도달하는 방식이 리눅스와 다르며, 
```

Comment:
[원문]
> Windows nodes do not overcommit memory for processes. The
net effect is that Windows won't reach out of memory conditions the same way Linux
does, and processes page to disk instead of being subject to out of memory (OOM)
termination. If memory is over-provisioned and all physical memory is exhausted,
then paging can slow down performance.

'condition' 이 1) 조건, 또는 2) 상태 로 해석될 수 있는데,
쿠버네티스에서는 'node condition' 이라고 하면 '노드의 상태' 를 나타내는 것으로 보이고

- https://kubernetes.io/ko/docs/concepts/architecture/nodes/#condition
> `conditions` 필드는 모든 `Running` 노드의 상태를 기술한다. **컨디션**의 예로 다음을 포함한다.

- https://kubernetes.io/ko/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions
> kubelet은 하드/소프트 축출 임계값 조건이 충족되어 노드 압박이 발생했다는 것을 알리기 위해, 설정된 유예 시간과는 관계없이 **노드 컨디션**을 보고한다.
> 
> kubelet은 다음과 같이 노드 컨디션과 축출 신호를 매핑한다.

이 부분에서도 '윈도우 노드가 OOM 상태에 도달하는 방식' 을 뜻하는 것으로 보여서
'컨디션' 으로 기재하였는데..
위의 사항에 대해서 어떻게 생각하시나요? 😊

Reply:
기존의 제 번역이 원문과 차이가 있어, 다음과 같이 수정했습니다. 😊

```
이로 인해 윈도우는 메모리 컨디션에 도달하는 방식이 리눅스와 다르며,
```